### ISSUE TITLE
CI fails due to missing dependency: kaszanas/sc2infoextractorgo

### ISSUE THREAD

**Issue by Ostrzyciel** [Kaszanas/SC2DatasetPreparator#29]
Currently the CI will fail on trying to pull the `kaszanas/sc2infoextractorgo` image, which it uses as a dependency. See for example here: https://github.com/Kaszanas/SC2DatasetPreparator/actions/runs/8467611697/job/23198781862

It seems that these images are not published anywhere. As there is no release with a Linux version of the executable, I can't easily circumvent this. To make the CI here usable we would have to either:
- a) implement Docker image building and publishing in sc2infoextractorgo for Docker Hub
- b) same, but with GitHub Container Registry (would be easier than Docker Hub)
- c) publish a Linux release and bodget it – instead of pulling the image, just run wget

In any case, I'd prioritize this, because the CI is otherwise useless.

**Comment by Kaszanas** [comment]
Yeah that's an issue for sure as I'm actively working on new release of the Golang piece of code which causes this issue.

It's worth noting that only one of the tools depends on `sc2infoextractorgo`.

It would be safe to state that the main Dockerfile inheriting from a non existing image can be ommited for the sake of verifying all of the other tools in the meantime.

I will try to push a docker image matching the version that was used to build the original dataset do this issue will be alleviated.

But I am quite sure that there will be done changes to that one `sc2_replaypack_processor` especially because some Golang flags have evolved.

**Comment by Kaszanas** [comment]
Okay @Ostrzyciel I have went and made up the missing pieces for you:
https://hub.docker.com/repository/docker/kaszanas/sc2infoextractorgo/general

I have made it by checking out the 1.2.1 tag which was the original release, patching in a `workflow_dispatch` manual trigger for building a docker image and running it.

My guess is that everything should be operational. I will experiment with building Docker images later. for separate `dev`, `main`, and release tags.
